### PR TITLE
Graph image type

### PR DESCRIPTION
**Pull request by HansenML**

### CODE REVIEW

File: lib/graph.js [hunk 1]
```diff
@@ -42,13 +42,35 @@ class Graph {
...
+            this.nodes.forEach(node => {
+              images.forEach(image => {
+                for (let type in image) {
+                  if (node.type === parseInt(type)) {
+                    node.image = image[type];
+                  }
+                }
+              });
+            });
+            // Add the image urls to each node depending on its type
```
Comment (RickyV33) [inline lib/graph.js]: Not sure what lines 65-72 are doing... Doesn't look like we need them since the work is already being done in lines  56-64

**Comment by RickyV33** [comment]
Removed some trailing spaces and added/removed some new lines in some places. That chunk of code still needs to be removed though.

File: lib/graph.js (hunk 1 above)
Comment (ruv-prog-so) [inline lib/graph.js]: Noted. Extra code removed.

**Comment by RickyV33** [comment]
LGTM

File: lib/graph.js [hunk 2]
```diff
@@ -72,7 +86,6 @@ class Graph {
 
   /**
    * Returns an object model abiding by the relationship graph JSON schema from the given nodes and edges.
-   *
```
Comment (chances) [inline lib/graph.js]: This is nice whitespace to have

**Comment by RickyV33** [comment]
LGTM

**Pull request merged**